tests/e2e/run.sh

```diff
@@ -43,6 +43,11 @@ test_e2e() {
     store_test_results "tests/all-tests-results" "all-tests-results"
     [[ ! -f FAIL ]] || die "e2e tests failed"
 
+    info "Sensor k8s integration tests"
+    make sensor-integration-test || touch FAIL
+    store_test_results "test-output/sensor-integration.log" "sensor-integration"
+    [[ ! -f FAIL ]] || die "e2e tests failed"
```

Comment:
Question here, wouldn't it make more sense to have a more nuanced output if it fails? eg.
`die "Sensor integration e2e tests failed"`